docs/02.API-REFERENCE.md

```diff
@@ -7692,6 +7716,67 @@ main (void)
 - [jerry_run](#jerry_run)
 - [jerry_vm_exec_stop_callback_t](#jerry_vm_exec_stop_callback_t)
 
+## jerry_set_promise_rejection_callback
+
+**Summary**
+
+Register a promise rejection tracker callback for the engine.
```

Comment:
We should provide more details here imho and also mention the `HostPromiseRejectionTracker` from the standard here.